app/styles/_data-toolbar.less

```diff
@@ -33,6 +33,12 @@
         font-size: @font-size-base;
       }
     }
+    .label-filter-key .selectize-input.not-full {
+      // Increase default input click target
+      width: 220px;
+      @media (min-width: @screen-xs) and (max-width: @screen-sm-max) { width: (@screen-xs - 100)} // 380px
```

Comment:
I don't think you need the `max-width` here since you have a rule just below that sets the width for `screen-md` and larger